cvat/apps/iam/permissions.py

```diff
@@ -21,10 +21,12 @@
 from cvat.apps.organizations.models import Membership, Organization
 from cvat.apps.engine.models import Project, Task, Job, Issue
 from cvat.apps.limit_manager.core.limits import (CapabilityContext, LimitManager,
-    Limits, OrgCloudStoragesContext, OrgTasksContext, ProjectWebhooksContext,
+    Limits, OrgCloudStoragesContext, OrgMembersContext, OrgTasksContext, OrgProjectWebhooksContext,
     OrgCommonWebhooksContext,
     TasksInOrgProjectContext, TasksInUserSandboxProjectContext, UserOrgsContext,
-    UserSandboxCloudStoragesContext, UserSandboxTasksContext)
+    UserSandboxCloudStoragesContext, UserSandboxProjectWebhooksContext,
+    UserSandboxTasksContext,
+    OrgLambdaCallOfflineContext, UserSandboxLambdaCallOfflineContext)
```

Comment:
Could you please change the import to `from cvat.apps.limit_manager.core import limits` so it was more clear? The list will only grow in future.